components/LocalList.php

```diff
@@ -80,6 +80,11 @@ protected function loadList()
 
         $this->mapIntoObjects($list);
 
+        if ($sortBy) {
+            $list->appends('sort_by', $sortBy);
+            $list->appends('pageLimit', $this->param('pageLimit'));
+        }
+
```

Comment:
Do we really need this? Why was it added?

Reply:
If you have the list sorted and there are more locations than the default page limit, clicking to the next page eliminates the sorting on the page. This change keeps the sorting when you go to page 2 and allows you to set the max # of locations on a single page.

Reply:
Thank you! I should have checked the Laravel docs instead, first time seeing the appends method. 

I've pushed some changes let me know if it's okay to merge.

Reply:
Looks good to me.